.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+<!--
+If you're submitting a bug report, it is helpful if you include the following information:
+
+- A minimal snippet of example code that demonstrates the issue.
+- The error code for the sniff that is being triggered (you can see the sniff error codes by running `phpcs` with the `-s` flag).
+- The versions of PHPCS, WPCS and PHP you are using.
+
+If you are submitting a feature describe what that feature would be and why it should be useful.
```

Comment:
This sentence feels a bit off. Maybe something like:
_If you are submitting a feature request, please describe the feature in detail and why it would be useful._

/cc @GaryJones